styles/namemc/catppuccin.user.css

```diff
@@ -0,0 +1,765 @@
+      --bs-btn-active-bg: @overlay0;
+      --bs-btn-active-border-color: @overlay0;
+      --bs-btn-disabled-color: @overlay0;
+      --bs-btn-disabled-border-color: @overlay0;
+    }
+    .btn-outline-success {
+      --bs-btn-color: @green;
+      --bs-btn-border-color: @green;
+      --bs-btn-hover-color: @text;
+      --bs-btn-hover-bg: @green;
+      --bs-btn-hover-border-color: @green;
+      --bs-btn-focus-shadow-rgb: @green;
+      --bs-btn-active-color: @text;
+      --bs-btn-active-bg: @green;
+      --bs-btn-active-border-colo: @green;
+      --bs-btn-disabled-color: @green;
+      --bs-btn-disabled-border-color: @green;
+    }
+    .btn-outline-info {
+      --bs-btn-color: @sapphire;
+      --bs-btn-border-color: @sapphire;
+      --bs-btn-hover-color: @crust;
+      --bs-btn-hover-bg: @sapphire;
+      --bs-btn-hover-border-color: @sapphire;
+      --bs-btn-focus-shadow-rgb: @sapphire;
+      --bs-btn-active-color: @crust;
+      --bs-btn-active-bg: @sapphire;
+      --bs-btn-active-border-color: @sapphire;
+      --bs-btn-disabled-color: @sapphire;
+      --bs-btn-disabled-border-color: @sapphire;
+    }
+    .btn-outline-warning {
+      --bs-btn-color: @yellow;
+      --bs-btn-border-color: @yellow;
+      --bs-btn-hover-color: @crust;
+      --bs-btn-hover-bg: @yellow;
+      --bs-btn-hover-border-color: @yellow;
+      --bs-btn-focus-shadow-rgb: @yellow;
+      --bs-btn-active-color: @crust;
+      --bs-btn-active-bg: @yellow;
+      --bs-btn-active-border-color: @yellow;
+      --bs-btn-disabled-color: @yellow;
+      --bs-btn-disabled-border-color: @yellow;
+    }
+    .btn-outline-danger {
+      --bs-btn-color: @red;
+      --bs-btn-border-color: @red;
+      --bs-btn-hover-color: @text;
+      --bs-btn-hover-bg: @red;
+      --bs-btn-hover-border-clor: @red;
+      --bs-btn-focus-shadow-rgb: @red;
+      --bs-btn-active-color: @text;
+      --bs-btn-active-bg: @red;
+      --bs-btn-active-border-color: @red;
+      --bs-btn-disabled-color: @red;
+      --bs-btn-disabled-border-color: @red;
+    }
+    .btn-outline-light {
+      --bs-btn-color: @text;
+      --bs-btn-border-color: @text;
+      --bs-btn-hover-color: @crust;
+      --bs-btn-hover-bg: @text;
+      --bs-btn-hover-border-color: @text;
+      --bs-btn-focus-shadow-rgb: @text;
+      --bs-btn-active-color: @crust;
+      --bs-btn-active-bg: @text;
+      --bs-btn-active-border-color: @text;
+      --bs-btn-disabled-color: @text;
+      --bs-btn-disabled-border-color: @text;
+    }
+    .btn-outline-dark {
+      --bs-btn-color: @surface0;
+      --bs-btn-border-color: @surface0;
+      --bs-btn-hover-color: @text;
+      --bs-btn-hover-bg: @surface0;
+      --bs-btn-hover-border-color: @surface0;
+      --bs-btn-focus-shadow-rgb: @surface0;
+      --bs-btn-active-color: @text;
+      --bs-btn-active-bg: @surface0;
+      --bs-btn-active-border-color: @surface0;
+      --bs-btn-disabled-color: @surface0;
+      --bs-btn-disabled-border-color: @surface0;
+    }
```

Comment:
this seems like a lot of repetition, it might be worth abstracting it into a mixin